### ISSUE TITLE
Fix `function-url-no-scheme-relative` false negatives for `@import`

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint/stylelint#8574]
### What is the problem you're trying to solve?

Disallowing scheme-relative urls for load rules could be an optional feature in this rule.
E.g.

```
@import url('//www.google.com/file.jpg')
```

### What solution would you like to see?

if `disallowInAtImport`is set to true then it applies to `@import` urls as well, or maybe this can be a separate rule?

**Comment by romainmenke** [comment]
Hi @pamelalozano16! Thank you for the report. I can reproduce the issue on [the online demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBLAtgBwgJwC4AIBXbAGwAoByAeioHd6A6AcwgiZJgcjSoDMUOXAM5CKASgDcAHQB2spLwgzcAWl4BDMDHzBZU3EOxh4RUpRr1azVu07c+AzrQi9e4-Iuxp1uMlJDOrgBM-pKyAL56MgzqOnq4AEaaANZM2BCEMlAmxOTUdIwsbIL2-IIAVhhM4tIy4SAANCBlMABy6mhwiDAAHh0YJSKN4Er8TAggujL4+P7YhBxC-iZTMzP+vJlguChKKrkqMhAqQmAAFjCdKtgwJD4oAG4wy-i488-T+JF1w5AyYwAxHDeXATcpCJTDWAYIQTVazEBCXAATw4JBQyhe-juuBgSP8DVk60RKLRGNUfzGJ1w6iy6mwUCxIBxeNw-giIHCQA).

I think this can be part of the existing rule as it is for functions in general.
I would even add this without an extra rule option to enable this behavior.

I think that anyone who wants disallow relative schemes will want to do so globally and not per construct (`@import` vs. decl)

I've labelled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
